README.md

```diff
@@ -126,7 +126,7 @@ The notes are sorted alphabetically, major (upper case) notes before minor
 | DDR4 DRAM          | [Corsair Vengeance RGB](docs/ddr4-guide.md) | SMBus | <sup>_Uax_</sup> |
 | DDR4 DRAM          | [Generic DDR4 temperature sensor](docs/ddr4-guide.md) | SMBus | <sup>_Uax_</sup> |
 | Fan/LED controller | [Corsair Commander Pro](docs/corsair-commander-guide.md) | USB HID | <sup>_h_</sup> |
-| Fan/LED controller | [Corsair Commander Core](docs/corsair-commander-core-guide.md) | USB HID | <sup>_ep_</sup> |
+| Fan/LED controller | [Corsair Commander Core, Core XT](docs/corsair-commander-core-guide.md) | USB HID | <sup>_ep_</sup> |
```

Comment:
Forgot one.

This _temporarily_ needs to be split into two entries, so that the one for the Core XT can be tagged with <sup>n</sup> ("Newly supported and requires git").

Reply:
Sure, done.